pandas/core/reshape/pivot.py

```diff
@@ -17,7 +17,7 @@
 
 from pandas.core.dtypes.cast import maybe_downcast_to_dtype
 from pandas.core.dtypes.common import is_integer_dtype, is_list_like, is_scalar
-from pandas.core.dtypes.generic import ABCDataFrame, ABCSeries
+from pandas.core.dtypes.generic import ABCDataFrame, ABCNDFrame, ABCSeries
```

Comment:
Series is in the namespace and there is an import from frame, might as well get DataFrame